core/src/consensus.rs

```diff
@@ -2286,10 +2283,6 @@ pub mod test {
         let mut new_votes = latest_validator_votes_for_frozen_banks.take_votes_dirty_set(0);
         new_votes.sort();
         assert_eq!(new_votes, account_latest_votes);
-
```

Comment:
voted_stake*lock_out is no longer used for choosing the forks. 
We no longer computes this.